src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
+
+.form {
+  position: relative;
+  margin: 0 auto;
+}
+
+.form::before {
+  content: '';
+  position: absolute;
+  z-index: 1;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.form_big::before {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.form_small::before {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+  transform: translateY(0%);
+}
+
+.input {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-top: 20px;
+  font-family: Avenir-Book, Arial, sans-serif;
```

Comment:
The `font-family` property should use the custom font name defined in the `@font-face` rule without fallback fonts. Use `Avenir` instead of `Avenir-Book`.